README.md

```diff
@@ -22,29 +22,28 @@ repositories {
 }
 
 dependencies {
-  compile 'com.github.paolorotolo:appintro:4.0.0'
+    compile 'com.github.paolorotolo:appintro:4.0.0'
 }
 ```
 
 Create a new **Activity that extends AppIntro**:
 
 ```java
-public class MyIntro extends AppIntro {
-
+public class IntroActivity extends AppIntro {
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
+    protected void onCreate(@Nullable Bundle savedInstanceState) {

```

Comment:
Don't annotate this just yet -- need to make a quick change on the library first.